wordpressorg.dev/provision/wp-config.php

```diff
@@ -10,6 +10,8 @@
 define( 'API_WPORGPATH',          __DIR__ . '/../../api.wordpress.org/public_html/includes/' );
 define( 'GLOTPRESS_LOCALES_PATH', __DIR__ . '/../../translate.wordpressorg.dev/public_html/glotpress/locales/locales.php' );
 define( 'WPORG_SANDBOXED',        true );
+define( 'PLUGINS_TABLE_PREFIX',   $table_prefix );
+define( 'WP_CORE_STABLE_BRANCH',  '4.5' );
```

Comment:
> move that to the top of the file

To the top of `sandbox-functionality.php`? Is that preferable over having it in `wp-config.php` with the other defines?